### PR TITLE
[#553] Implement weekly dependency check job

### LINKED ISSUE #553: Add weekly dependency check job

### PR DESCRIPTION
**Pull request by oanamariabalaita**
Resolve #553

### CODE REVIEW

File: .github/workflows/dependencies.yml [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+            						"text": '${{ env.depss }}'.split('%0A').join('\n').split('%0D').join('\r')
+            					}
+            				}
+            			]
+            		}
+            	]
+            }
+        env:
+          SLACK_BOT_TOKEN: ${{ secrets.SLACK_BOT_TOKEN }}
+        if: always()
```
Comment (mustafaozhan) [inline .github/workflows/dependencies.yml:76]: let's set the job status here

```yml
      - name: Set Job Status
        id: status
        run: echo "::set-output name=status::success"
```

File: .github/workflows/dependencies.yml [hunk 2]
```diff
@@ -0,0 +1,92 @@
+name: Check dependency updates
```
Comment (mustafaozhan) [inline .github/workflows/dependencies.yml:1]: let's Capitalise the name also 🙂

File: .github/workflows/dependencies.yml (hunk 1 above)
Comment (oanamariabalaita) [reply .github/workflows/dependencies.yml:76]: Applied in e5a7db9daa05b0fa7397a2c95bd52c6d5e97d29c

File: .github/workflows/dependencies.yml (hunk 2 above)
Comment (oanamariabalaita) [reply .github/workflows/dependencies.yml]: Sure, done in e5a7db9daa05b0fa7397a2c95bd52c6d5e97d29c 🙂

**Review by mustafaozhan — APPROVED** [review]

**Pull request merged**